src/Flow/ETL/Stream/Handler.php

```diff
@@ -33,7 +33,7 @@ public function open(FileStream $stream, Mode $mode)
     {
         /** @psalm-suppress PossiblyNullOperand */
         $fullPath = ($this->safeMode)
-            ? (\rtrim($stream->uri(), DIRECTORY_SEPARATOR) . DIRECTORY_SEPARATOR . \uniqid() . '.' . $this->extension)
+            ? (\rtrim($stream->uri(), DIRECTORY_SEPARATOR) . DIRECTORY_SEPARATOR . \uniqid('', true) . '.' . $this->extension)
```

Comment:
one thing I would consider here is removing additional dot, output of uniqid with second parameter set to true would be something like this `62b89bc101a5f7.27307237`.
With addition of the extension it would become `62b89bc101a5f7.27307237.csv` for example which is not terrible but `62b89bc101a5f727307237.csv` would look a little bit better.

Reply:
Replaced with: https://www.php.net/manual/en/function.random-bytes.php

Later we can move all of them to that function.